modules/gapi/include/opencv2/gapi/own/assert.hpp

```diff
@@ -18,6 +18,7 @@
 #if !defined(GAPI_STANDALONE)
 #include <opencv2/core/base.hpp>
 #define GAPI_Assert CV_Assert
+#define GAPI_Error CV_Error
```

Comment:
There should be a STANDALONE version defined, too.

Reply:
To define GAPI_Error for the Standalone mode, it's necessary to create a separate task and in framework the task to add Error logging. This changes should place to a separate PR.

Reply:
For the gfluidimgproc.cpp it is permissible to use OCV error login system, since this file is under  `#if !defined(GAPI_STANDALONE)` guard.

Reply:
> For the gfluidimgproc.cpp it is permissible to use OCV error login system, since this file is under [#if !defined(GAPI_STANDALONE)](https://github.com/opencv/opencv/blob/ef6f421f8953c0a9d4e5d49074b0dc99975a3fa0/modules/gapi/src/backends/fluid/gfluidimgproc.cpp#L7) guard.

Then just drop `GAPI_Error`, use `CV_Error` in your code, and file a task to implement one.

Half-defining this macro here in assert only brings confusion (as it is generic and may be used elsewhere, not only at `gfluidimgproc.hpp`)